content/blog/2023/when-to-choose-a-progressive-web-app.md

```diff
@@ -0,0 +1,123 @@
+
+Since PWAs are a type of web app, search engines like Google will automatically index them and include them in search results. Various tools such as PWABuilder essentially wrap up a PWA like a native app for inclusion in the Google, Apple, and Windows app stores.
+
+### Broad Reach
+*…when you want to reach a broad audience across devices.*
+
+If budget allows, PWAs can reach a broader audience than either responsive web apps or native mobile apps. With a single codebase, people can access the RWA on any device with a browser. They can find it from a search engine and, with some extra packaging, in any app store. They can access a PWA via the internet or download it onto their phone like a native mobile app and use it offline. Like RWAs, PWAs take advantage of the web's high level of "backward compatibility," meaning people on older devices will have some level of access.
+
+### Specialize Capabilities
+*…when you want to reach a broad range of people while keeping costs down as much as possible, but need some specialized capabilties.*
+
+PWAs are better than native mobile apps or RWAs at providing a balance between reach, cost, and specialization. If you need to be able to send alerts or "push notifications" to people even when they do not have the app open on their phone, PWAs can do that just like native mobile apps, whereas RWAs cannot.
+
+If you cannot afford to build and maintain more than one codebase, but people using your app may have limited or sometimes no access to the internet, a PWA is likely the best option. While it may not perform as seamlessly as a native app without internet access, people can download a PWA onto their phone and access the content without a wifi or data connection.
+
+Accessing an RWA requires opening a browser, but people can access a PWA via an icon on their phone's homescreen just like they do with a native mobile app. This is not just a shortcut to a website, but rather a standalone application completely independent from the browser interface.
+
+Check out and test all the PWA capabilities on [What PWA Can Do Today](https://whatpwacando.today/).
+
+{{ embed.figure(
+  data=[{
+    img: 'blog/2023/no.png',
+    alt: 'The word no with a frowning face'
+  }],
+  attrs={
+    'style': '--column: content-start'
+  }
+) }}
+
+## Don’t choose a PWA when…
+
+In a nutshell, if your budget and timeline is quite constrained and you don't need any special capabilities, then the best option would be to build a responsive web app instead of a PWA. After you have more information about what the people using your app want and you have additional budget, you can always turn the RWA into a PWA. On the otherhand, if you do have plenty of budget and highly specialized needs that require full integration with a specific mobile device, a native mobile is going to be a better option than a PWA.
```

Comment:
```suggestion
In a nutshell, if your budget and timeline is quite constrained and you don’t need any special capabilities, then the best option would be to build a responsive web app instead of a PWA. After you have more information about what the people using your app want and you have additional budget, you can always turn the RWA into a PWA. On the other hand, if you do have plenty of budget and highly specialized needs that require full integration with a specific mobile device, a native mobile is going to be a better option than a PWA.
```